site/smart-accounts/accounts/light-account.md

```diff
@@ -14,35 +14,49 @@ head:
 
 # Light Account
 
-Light Account is a secure, gas-optimized, [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
+### Secure, optimized, and extendable
 
-We started with the Ethereum Foundation’s canonical [SimpleAccount](https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol) and added key improvements for production app developers:
+In Web3, efficiency, trust, and security are non-negotiable, especially with your account. That's why we built Light Account! With Light Account, we’ve turbocharged Ethereum smart accounts, reducing gas costs and supporting ERC-1271 signatures to add portability amongst all dApps. Plus, we've got your back with ownership transfer, ensuring you're never locked into a single signer. Our benchmark results speak for themselves, giving you the confidence to use a fully audited, optimized, and [open-sourced](https://github.com/alchemyplatform/light-account) smart account solution.
```

Comment:
this paragraph is trying to do too many things at once. Let's structure this doc into discrete sections for clarity
* Overview:  it's important for docs to start at square one. First section should simply define what Light Account is. I think we should keep the section that was here previously, and then add what you wrote as a "Why Light Account" section explaining the benefits. 
* Why Light Account
* * Gas-optimized (benchmarks)
* * Secure (audited, etc)
* * Coming soon: Modular

Copy and voice notes:
* web3 lowercase
* turbocharged is hyperbolic
* 'we've got your back" is a bit too casual / personifying the contract
* "Our benchmark results speak for themselves, giving you the confidence" also too casual. Goal is to follow the [Google style guidelines](https://developers.google.com/style/tone)